skills/investigate/ingest/scripts/ingest.py

```
@@ -539,13 +550,187 @@ def collapse_repeats(segs):
+        return [], 0
+    meta, words, failed = [], 0, 0
+    # Splice back-to-front so earlier spans' indices stay valid.
+    for i, j in reversed(spans):
+        w_start = max(0.0, cues[i][0] - RECOVERY_PAD_S)
+        w_end = cues[j][1] + RECOVERY_PAD_S
+        new = retranscribe_span(wav, w_start, w_end, out_dir)
+        if new is None:
+            failed += 1
+            log(
+                f"WARNING: loop-wedge recovery FAILED for "
+                f"{cues[i][0]:.0f}s–{cues[j][1]:.0f}s — original cues kept; "
```

Ruff (0.16.1)

[warning] String contains ambiguous `–` (EN DASH). Did you mean `-` (HYPHEN-MINUS)?

(RUF001)